documentation.html

```diff
@@ -154,6 +154,125 @@ <h5>Examples of Outline Alert Box:</h5>
                     </iframe>
             </div>
             <!-- ALert Component Ends  -->
+            <!-- Badge Comonent Starts  -->
+            <div class="badge-component">
+                <h3>Badges</h3>
+                <p>Badges are being used to display a notification count or status information.</p>
+                <p>We have two types of avatar badges that tells whether the user is online, offline or in DND mode.
+                    Along with it icon badges are also present.</p>
+                <h5>Examples of Dot Profile Badges:</h5>
+                <div class="container">
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-m" src="./assets/avatar.png" alt="Avatar" />
+                        <div class="badge-dot badge-online-a"></div>
+                    </div>
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-m" src="./assets/avatar.png" alt="Avatar" />
+                        <div class="badge-dot badge-offline-a"></div>
```

Comment:
badge-online-a, badge-online-b these names having a,b does not make much sense, please add more meaningful names instead.